_unittest/test_21_Circuit.py

```diff
@@ -80,9 +80,26 @@ def test_05c_create_component(self):
             "Test1", [1, 2], parameter_list=["Author:=", "NumTerminals:="], parameter_value=["pyaedt", 2]
         )
 
-    def test_06_add_3dlayout_component(self):
+    def test_06a_create_setup(self):
+        setup_name = "LNA"
+        LNA_setup = self.aedtapp.create_setup(setup_name)
+        assert LNA_setup
```

Comment:
@Samuelopez-ansys I think that the variable is not mandatory here.
```suggestion
        assert self.aedtapp.create_setup(setup_name)
```